packages/hadron-express/index.ts

```diff
@@ -16,11 +19,22 @@ import {
 
 export default hadronExpress;
 
-export const register = (container: IContainer, config: IHadronExpressConfig) =>
-  hadronExpress(
+export const register = (
+  container: IContainer,
+  config: IHadronExpressConfig,
+) => {
+  const port = process.env.PORT || 8080;
+  const expressApp = express();
+  expressApp.use(bodyParser.json());
+  container.register('server', expressApp);
+  container.register('serverListen', () => {
+    expressApp.listen(port);
```

Comment:
This could be executed maybe after all packages would load (maybe using hadron-events package) or there could be an option in config's objects to disable auto-executing listen method and do it manually - with that option there should be a possibility to provide more custom parameters (https://expressjs.com/en/api.html#app.listen).